controllers/__init__.py

```diff
@@ -276,14 +275,16 @@ def upsert_interface_details(
         self,
         id_: str,
         available_tags: dict[str, list[list[int]]],
-        tag_ranges: dict[str, list[list[int]]]
+        tag_ranges: dict[str, list[list[int]]],
+        special_available_tags: dict[str, list[str]]
     ) -> Optional[dict]:
         """Update or insert interfaces details."""
         utc_now = datetime.utcnow()
         model = InterfaceDetailDoc(**{
                 "_id": id_,
                 "available_tags": available_tags,
                 "tag_ranges": tag_ranges,
+                "special_available_tags": special_available_tags,
```

Comment:
I wonder if we should also take the opportunity to also store, `special_tag_ranges` https://github.com/kytos-ng/kytos/pull/431#discussion_r1408117896, that way it's also aligned with how `tag_ranges` is being set, except for the special values, wdyt?

Reply:
That said, currently, there's no way via API to know which special tags or special tag ranges are configured, except running Python code on `kytosd` console. I wonder if we should also expose those in the same endpoints, but with `special_tag_ranges` and `special_available_tags` and their values, currently we have:

```
❯ curl -s http://0.0.0.0:8181/api/kytos/topology/v3/interfaces/tag_ranges | jq
{
  "00:00:00:00:00:00:00:01:4294967294": {
    "available_tags": {
      "vlan": [
        [
          1,
          3798
        ],
        [
          3800,
          4095
        ]
      ]
    },
    "tag_ranges": {
      "vlan": [
        [
          1,
          4095
        ]
      ]
    }
  },

```

Reply:
`special_tag_range` and `special_available_tags` are now included in the returned dictionary, Also new api request to modify `special_tag_range` was added, uni tests are missing.